### PR TITLE
Fix dates source on UsageReport page

### PR DESCRIPTION
**Pull request by rikysya**
What: fix dates source on usage report page

Why: to fix https://storjlabs.atlassian.net/browse/V3-1831

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: web/satellite/src/components/project/UsageReport.vue [hunk 1]
```diff
@@ -161,13 +161,15 @@ import { toUnixTimestamp } from '@/utils/time';
                     });
                 },
                 onReportClick: function (): void {
-                    let projectID = this.$store.getters.selectedProject.id;
+                    const projectID = this.$store.getters.selectedProject.id;
+                   	const startDate = this.$store.state.usageModule.startDate;
+					const endDate = this.$store.state.usageModule.endDate;
```
Comment (stefanbenten) [inline web/satellite/src/components/project/UsageReport.vue]: The Idents look off to me

Comment (ifraixedes) [reply web/satellite/src/components/project/UsageReport.vue]: isn't there a linter for this part of the code which could check this kind of things?

Comment (rikysya) [reply web/satellite/src/components/project/UsageReport.vue]: Yeah, that is something with my IDE it shows idents in a correct place even though they are not)
There is one but it isn't integrated with travis

Comment (ifraixedes) [reply web/satellite/src/components/project/UsageReport.vue]: Let's open an issue in Jira for it with low priority

**Pull request merged**